data/korath/korath outfits.txt

```diff
@@ -224,7 +224,7 @@ outfit "Afterburner (Asteroid Class)"
 	"afterburner heat" 6.25
 	"afterburner fuel" 0.0375
 	"afterburner effect" "korath afterburner tiny"
-	description "The smallest of afterburners is as much fuel processor as it is afterburner. It allows Korath small craft operators to squeeze a hyperdrive into otherwise intrasystem ships and dispense with conventional thrusters entirely."
+	description "The smallest of afterburners is as much Fuel Processor as it is afterburner. It allows Korath small craft operators to squeeze a hyperdrive into otherwise intrasystem ships and dispense with conventional thrusters entirely."
```

Comment:
Another great example of how what wouldn't otherwise be a proper reference becomes one because of the functionalist naming scheme. :)